azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -544,7 +549,9 @@ private void prepareAndStartServer()
     createThreadPool();
     configureRoutes();
 
-    if (this.props.getBoolean(Constants.ConfigurationKeys.IS_METRICS_ENABLED, false)) {
+    // Todo jamiesjc: enable web metrics for azkaban poll model later
```

Comment:
Aside: some people like to add date (along with the author name) to their todos, so that readers can identify which era the todo comes from.

Reply:
That's a good suggestion. Currently, I'm just following our existing style guide in CONTRIBUTING.md:  
 
>  todo username: todo details